LICENSE

```diff
@@ -1,7 +1,7 @@
-Copyright (c) 2016, Los Alamos National Security, LLC
+Copyright (c) 2017, Los Alamos National Security, LLC
```

Comment:
My thinking was, 2016 is when LANL took over and hence the copyright holds from 2016-now (2017).